RELEASING.md

```diff
@@ -0,0 +1,47 @@
+# Exposure Notifications Verificion Servers Release Process
```

Comment:
Instead of duplicating the docs, should we just link to one so we don't have config drift?

Reply:
updated